docs/examples/floating-pv/plot_floating_pv_cell_temperature.py

```diff
@@ -0,0 +1,217 @@
+| closed structure,       |             |                                |                                        |           |
+| large water footprint   |             |                                |                                        |           |
++-------------------------+-------------+--------------------------------+----------------------------------------+-----------+
+| Monofacial module,      | Singapore   | 18.9                           | 8.9                                    | [1]_      |
+| closed stucuture,       |             |                                |                                        |           |
+| medium water footprint  |             |                                |                                        |           |
++-------------------------+-------------+--------------------------------+----------------------------------------+-----------+
+| Monofacial module,      | Singapore   | 35.3                           | 8.9                                    | [1]_      |
+| open strucuture,        |             |                                |                                        |           |
+| free-standing           |             |                                |                                        |           |
++-------------------------+-------------+--------------------------------+----------------------------------------+-----------+
+| Monofacial module,      | Norway      | 86.5                           | 0                                      | [2]_      |
+| in contact with         |             |                                |                                        |           |
+| water                   |             |                                |                                        |           |
++-------------------------+-------------+--------------------------------+----------------------------------------+-----------+
+| Monofacial module,      | South Italy | 31.9                           | 1.5                                    | [3]_      |
+| open structure,         |             |                                |                                        |           |
+| free-standing           |             |                                |                                        |           |
++-------------------------+-------------+--------------------------------+----------------------------------------+-----------+
+| Bifacial module,        | South Italy | 35.2                           | 1.5                                    | [3]_      |
+| open structure,         |             |                                |                                        |           |
+| free-standing           |             |                                |                                        |           |
++-------------------------+-------------+--------------------------------+----------------------------------------+-----------+
```

Comment:
```suggestion
.. table:: Heat transfer coefficients for different PV systems
   :widths: 40 15 15 15 15

   +--------------------------+-------------+--------------------------------+----------------------------------------+-----------+
   | System                   | Location    |:math:`U_{c}`                   | :math:`U_{v}`                          | Reference |
   |                          |             |:math:`[\frac{W}{m^2 \cdot K}]` | :math:`[\frac{W}{m^3 \cdot K \cdot s}]`|           |
   +==========================+=============+================================+========================================+===========+
   | - Monofacial module      | Netherlands | 24.4                           | 6.5                                    | [1]_      |
   | - Open structure         |             |                                |                                        |           |
   | - Two-axis tracking      |             |                                |                                        |           |
   | - Small water footprint  |             |                                |                                        |           |
   +--------------------------+-------------+--------------------------------+----------------------------------------+-----------+
   | - Monofacial module      | Netherlands | 25.2                           | 3.7                                    | [1]_      |
   | - Closed structure       |             |                                |                                        |           |
   | - Large water footprint  |             |                                |                                        |           |
   +--------------------------+-------------+--------------------------------+----------------------------------------+-----------+
   | - Monofacial module      | Singapore   | 34.8                           | 0.8                                    | [1]_      |
   | - Closed structure       |             |                                |                                        |           |
   | - Large water footprint  |             |                                |                                        |           |
   +--------------------------+-------------+--------------------------------+----------------------------------------+-----------+
   | - Monofacial module      | Singapore   | 18.9                           | 8.9                                    | [1]_      |
   | - Closed structure       |             |                                |                                        |           |
   | - Medium water footprint |             |                                |                                        |           |
   +--------------------------+-------------+--------------------------------+----------------------------------------+-----------+
   | - Monofacial module      | Singapore   | 35.3                           | 8.9                                    | [1]_      |
   | - Open structure         |             |                                |                                        |           |
   | - Free-standing          |             |                                |                                        |           |
   +--------------------------+-------------+--------------------------------+----------------------------------------+-----------+
   | - Monofacial module      | Norway      | 86.5                           | 0                                      | [2]_      |
   | - In contact with water  |             |                                |                                        |           |
   +--------------------------+-------------+--------------------------------+----------------------------------------+-----------+
   | - Monofacial module      | South Italy | 31.9                           | 1.5                                    | [3]_      |
   | - Open structure         |             |                                |                                        |           |
   | - Free-standing          |             |                                |                                        |           |
   +--------------------------+-------------+--------------------------------+----------------------------------------+-----------+
   | - Bifacial module        | South Italy | 35.2                           | 1.5                                    | [3]_      |
   | - Open structure         |             |                                |                                        |           |
   | - Free-standing          |             |                                |                                        |           |
   +--------------------------+-------------+--------------------------------+----------------------------------------+-----------+
```

Formats columns widths so description fits in lower cells and bullet-lists the characteristics for inspecting it more easily.
Fixes typos `stucuture` & `strucuture`.
Feel free to modify as needed, I haven't built it so problems may arise.